app/api/routes/route.ts

```
@@ -0,0 +1,45 @@
+    const files = fs.readdirSync(llmContentDir);
+
+    // Convert file names to routes
+    const routes = files.map((file) => {
+      // Remove .md extension and convert hyphens to slashes
+      const baseName = file.replace('.md', '');
+
+      // Handle special cases for weekly updates
+      const updatePattern = /updates-year-\d{4}-week-\d{1,2}/;
+      if (updatePattern.exec(baseName)) {
+        // Convert updates-year-2025-week-11.md to /llms/updates/year/2025/week/11
+        return '/llms/' + baseName.replace(/-/g, '/');
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)